spec/mailers/previews/user_mailer_preview.rb

```diff
@@ -181,7 +181,7 @@ def in_person_enrollment
         enrollment_code: '2048702198804358',
         created_at: Time.zone.now - 2.hours,
         status_updated_at: Time.zone.now - 1.hour,
-        current_address_matches_id: true,
+        current_address_matches_id: params['current_address_matches_id'] == 'true',
```

Comment:
I forgot to mention this in the original pull request. Previously, the preview would omit the "Bring proof of address" content, consistent with the expected behavior when the current address matches the ID. This flips the default, but also allows for it to be hidden by adding a query parameter `?current_address_matches_id=false`.